frontend/src/photographerView/photographerView.scss

```diff
@@ -0,0 +1,27 @@
+.photographer-name {
```

Comment:
Rather than manually set each of these as a different font-size in the CSS, we should use semantic HTML. e.g., `<div className='photographer-name'>` above should probably be an `<h1>`, the individual things like attributes could be `<h2>` etc.

See: https://webaim.org/techniques/semanticstructure/ for why -- it's both good for accessibility (lets devices like screen readers understand the structure of the page) and means we can setup universal heading styles across the project if we don't like the built-in bootstrap styles.

Reply:
Same deal applies to the attributes in the PhotoView, btw.